src/components/OpenSourceBlurb.astro

```diff
@@ -0,0 +1,73 @@
+---
+import { Image } from "astro:assets";
+import gradientLineImg from "../images/gradient-line.svg";
+import discordIllustrationImg from "../images/discord-illustration.svg";
+import branchingImg from "../images/branching.svg";
+import heartImg from "../images/heart.svg";
+import Card from "./Card.astro";
+---
+
+<section class="container mx-auto pt-16 lg:py-32 px-4">
+  <div class="max-w-fit mx-auto mb-5 lg:mb-16 flex flex-col items-center gap-2">
+    <Image src={gradientLineImg} alt="A gradient line." class="mb-2" />
+    <h1 class="font-bold text-3xl">
+      Grain
+      <Image src={heartImg} alt="A heart." class="inline" />
+      open source
+    </h1>
+    <p>Grain is a passion project. All of our projects are <a href="https://github.com/grain-lang" class="text-color-accent underline hover:opacity-80">open source</a>.</p>
+  </div>
+  <div class="mx-auto">
+    <div class="grid grid-cols-1 lg:grid-cols-2 gap-5 lg:gap-14">
+      <Card title="We continuously strive to" class="bg-color-background-bright">
+        <ul class="pl-6 [&>li]:my-1 [&>li]:pl-1 text-color-accent list-outside list-image-[url(/src/images/check-icon.svg)]">
```

Comment:
This is a NIT but I found the contrast of the orange here didn't quite feel right, almost as though we should make it brighter, or maybe even make the text thiner.